tfgcv/planned_assets.go

```diff
@@ -44,40 +41,14 @@ func ReadPlannedAssets(ctx context.Context, path, project, ancestry string, offl
 
 	var resources []tfplan.Resource
 
-	switch version.LeastSupportedVersion() {
-	case version.TF12:
-		data, err := readTF12Data(path)
-		if err != nil {
-			return nil, err
-		}
-
-		resources, err = tfplan.ComposeTF12Resources(data, converter.Schemas())
-		if err != nil {
-			return nil, errors.Wrap(err, "unmarshal from JSON and composing terraform plan")
-		}
-	case version.TF11:
-		f, err := os.Open(path)
-		if err != nil {
-			return nil, errors.Wrap(err, "opening plan file")
-		}
-		defer f.Close()
-
-		plan, err := terraform.ReadPlan(f)
-		if err != nil {
-			return nil, errors.Wrap(err, "reading terraform plan")
-		}
-
-		// Attempt to pull the project from the provider.
-		if project == "" {
-			project, err = parseProviderProject(plan)
-			if err != nil {
-				return nil, err
-			}
-		}
+	data, err := readTF12Data(path)
```

Comment:
What happens if the user is still running 0.11? Will this throw an informative error?

Reply:
In TF 0.11 validator read the plan file directly; in TF 0.12 it reads a json file. When you try to pass in a non-JSON file you get:

```
Error: converting tfplan to CAI assets: Terraform 0.12 support plans only in JSON format, got: .tfplan
```